### DISCUSSION TITLE
Copying the temp plot went wrong, is it safe to rename?

### DISCUSSION THREAD

**Discussion by Neosettler** [swar/Swar-Chia-Plot-Manager#299] [Q&A]
When the copy has been interrupted, is it safe to rename .plot.2.tmp to .plot and copy the file manually?

**Comment by luluu9** [comment]
You can try to do this. If plot is corrupted, it will be shown under "Corrupted plots" or something like this in Chia's GUI.

**Comment by Neosettler** [comment]
The GUI seems fine. I was under the impression that there is a command line to validate plots... I'm not sur if "chia plots check" does much in that regard.

**Comment by swar** [comment]
Please use GitHub Discussions for questions, this is not an issue.
